test/parallel_api/experimental/radix_sort_esimd.pass.cpp

```diff
@@ -192,9 +192,9 @@ int main()
         for(auto size: sizes)
         {
             test_general_cases<uint32_t>(size);
-            // test_general_cases<int>(size);
-            // test_general_cases<float>(size);
-            // test_general_cases<double>(size);
+            test_general_cases<int>(size);
+            test_general_cases<float>(size);
+            test_general_cases<double>(size);
```

Comment:
Let's comment it out, because the current implementation does not support `double` type. For example, it uses [gather](https://intel.github.io/llvm-docs/doxygen/group__sycl__esimd__memory.html#gaf7753dcd28985606893834239883c6d5) instruction, which has the following restriction:
> Tx	Element type, must be of size 4 or less.

Reply:
That's a pretty severe limitation; we need to understand how we address it later.